ios/Example/Tests/CertificateDecodableTest.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CertificateDecodableTest.swift
+//  SecuritySdk_Tests
+//
+
+import Foundation
+import SecuritySdk
+import XCTest
+
+final class CertificateDecodableTest: XCTestCase {
+    /// signed device certificate signed by JVM SDK
```

Comment:
Something that always helps me later is the command/test name that was used to generate this data. Can you please add it to both of your test data?

Reply:
ran the unit test to print the `.pem()` call under `PKCS10CertificationRequestTests` test class